ruby_on_rails/app_initialisation.md

```diff
@@ -1,32 +1,42 @@
 # Initialise the Rails App
 
-* [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not. You can do this with `gem update rails`.
+* Check if you are using the latest version of ruby supported by SemahoreCI and Heroku with `ruby -v` and update it if you are not.
+Run `rbenv install -s` to check your installed ruby versions.
+Follow the instructions on screen to eventually install it.
 
-* Start a new Rails project using the usual `rails new -d postgresql [project-name]` where the `project-name` is exactly the one you chose before.
+* Run `gem install bundler` to install Bundler.
+You may have it already installed, but this command will make sure that you have the latest version.
+
+* [Check if you are using the latest stable version of Rails](http://rubyonrails.org/) with `rails -v` and update it if you are not.
+You can do this with `gem update rails`.
+
+* Start a new Rails project using `rails new [project-name] -d postgresql --skip-sprockets --webpack` where the `project-name` is exactly the one you chose before.
 You may want to choose a different database from Postgres, but most of the time that will be your choice.
 If you do not need a DB you may rethink the fact that you may not need Rails at all :) Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)
 
-* Add `.idea` to the `.gitignore` file since many of us use RubyMine.
+* Specify the ruby version in the project's `Gemfile` with
 
-* Add `tags` to the `.gitignore` file since many of us use CTags.
+```ruby File.read(File.join(__dir__, '.ruby-version'))```
```

Comment:
by default the `bundle` command can be run in any subfolder. if you use `File.read('.ruby-version')`, the `bundle` command will fail because it looks for the `.ruby-version` file in the current folder. This solves the issue and searches the file in the same folder of the `Gemfile`.